desktop/assets/texts/es-ES/preferences_dialog.ftl

```diff
@@ -28,5 +28,5 @@ gamemode = GameMode
 gamemode-tooltip =
     GameMode aplica temporalmente un set de optimizaciones a tu computadora y/o a Ruffle.
     Ruffle solicita GameMode solo cuando se está reproduciendo una película.
-gamemode-default = Por defecto
-gamemode-default-tooltip = GameMode solo será habilitado cuando la preferencia de poder esté en alto.
+gamemode-default = gamemode-predeterminado
```

Comment:
This is definitely not right...

Reply:
They translated the key instead of translating the value...

Reply:
Undid it on Crowdin.